Drift/Managers/SocketManager.swift

```diff
@@ -83,24 +84,32 @@ class SocketManager {
     }
     
     func computeShardId(orgId: Int) -> Int{
-        return orgId % 50 //WS_NUM_SHARDS
+        return orgId % 50
     }
     
     func willReconnect() {
-        NotificationCenter.default.post(name: .driftSocketStatusUpdated, object: self, userInfo: ["connectionStatus": ConnectionStatus.connecting])
+        DispatchQueue.main.async {
```

Comment:
Is the idea that these notifications update the UI and need to be on the main thread?

Reply:
Yeah exactly, my expectation would be that NSNotifications would always be on the main treat but happy to do it differently if you have another option?

Reply:
I was just curious